bokehjs/package.json

```diff
@@ -80,24 +80,25 @@
     "@typescript-eslint/parser": "^2.24.0"
   },
   "dependencies": {
-    "tslib": "^1.11.0",
     "@bokeh/canvas2svg": "^1.0.22",
+    "@bokeh/gloo2": "^0.0.1",
+    "@bokeh/numbro": "^1.6.2",
+    "@bokeh/slickgrid": "^2.3.23",
+    "choices.js": "^9.0.1",
     "es5-ext": "^0.10.53",
-    "es6-promise": "4.2.8",
-    "es6-weak-map": "^2.0.2",
     "es6-map": "^0.1.5",
+    "es6-promise": "4.2.8",
     "es6-set": "^0.1.5",
+    "es6-weak-map": "^2.0.2",
     "flatbush": "^3.2.1",
     "flatpickr": "^4.6.3",
-    "@bokeh/gloo2": "^0.0.1",
     "hammerjs": "^2.0.4",
+    "nan": "^2.14.0",
     "nouislider": "^10.0.0",
-    "@bokeh/numbro": "^1.6.2",
     "proj4": "^2.6.0",
-    "choices.js": "^9.0.1",
-    "@bokeh/slickgrid": "^2.3.23",
     "sprintf-js": "^1.1.2",
     "timezone": "^1.0.23",
+    "tslib": "^1.11.0",
```

Comment:
Please revert unnecessary changes.

Reply:
Did you add `nan` package intentionally? I will assume not. In that case my advice is to use `git add -p` (or `--patch`) to avoid committing unintentional changes.

Reply:
I will remove these files.  

I wasn't sure whether or not to included them in my commit but as they were showing up as modified files I decided to included them. I kinda suspected that was the wrong move but wasn't sure.

As to why they were there in the first place.  When following the instructions at ```https://docs.bokeh.org/en/latest/docs/dev_guide/setup.html#devguide-setup```
In the section for installing node package when running ```npm ci```
I got the following error ```cannot find module 'nan'```

running ```npm install nan``` then running ```npm ci``` fixed the issue.